client/my-sites/stats/geochart/index.jsx

```diff
@@ -18,17 +17,51 @@ module.exports = React.createClass( {
 	visualization: null,
 
 	propTypes: {
-		data: React.PropTypes.array.isRequired,
-		dataList: React.PropTypes.object.isRequired
+		data: PropTypes.array.isRequired,
+		dataList: PropTypes.object.isRequired
```

Comment:
This is a pattern I picked up from @aduth - just a shortcut really instead of typing out React.PropTypes.  Thanks for the review by the way!